Content.Server/GameTicking/GameTicker.cs

```diff
@@ -214,7 +215,35 @@ public void EndRound()
 
             RunLevel = GameRunLevel.PostRound;
 
-            SendServerMessage("The round has ended!");
+            //Tell every client the round has ended.
+            var roundEndMessage = _netManager.CreateNetMessage<MsgRoundEndMessage>();
+            roundEndMessage.GamemodeTitle = MakeGamePreset().ModeTitle;
+
+            //Get the timespan of the round.
+            var gameTime = IoCManager.Resolve<IGameTiming>();
+            roundEndMessage.RoundDuration = gameTime.RealTime;
+
+            //Generate a list of basic player info to display in the end round summary.
+            var listOfPlayerInfo = new List<RoundEndPlayerInfo>();
+            foreach(var ply in _playerManager.GetAllPlayers().OrderBy(p => p.Name))
+            {
+                if (ply == null) continue;
+                if(ply.AttachedEntity.TryGetComponent<MindComponent>(out var mindComponent)
+                    && mindComponent.HasMind)
+                {
+                    var playerEndRoundInfo = new RoundEndPlayerInfo()
+                    {
+                        PlayerOOCName = ply.Name,
+                        PlayerICName = mindComponent.Mind.CurrentEntity.Name,
+                        Role = mindComponent.Mind.AllRoles.First().Name,
```

Comment:
I know this is unlikely, but it's probably best to have a better-safe-than-sorry in case somebody has no roles.